cluster-http/src/main/resources/reference.conf

```diff
@@ -24,4 +20,14 @@ akka.management {
       ready-states = ["Up", "WeaklyUp"]
     }
   }
```

Comment:
should we remove the `http` part for above `ready-states` config?

Reply:
Yep